scripts/gitlab/build_and_test.sh

```diff
@@ -137,9 +137,17 @@ then
 
     cd ${build_dir}
 
-    date
-    ctest --output-on-failure -T test 2>&1 | tee tests_output.txt
-    date
+    # If HIP enabled 
+    if [[ "${option}" != "--build-only" ]] && grep -q -i "ENABLE_HIP.*ON" ${hostconfig_path}
+    then #don't run the tests that are known to fail
+        date
+        ctest --output-on-failure -T test -E Known-Hip-Failure 
```

Comment:
What happens if there is an unknown hip failure? I'm afraid that as-is, corona test may never exit in error in gitlab.
You can see `line 148` that I am storing results in `tests_outputs.txt` so that I can then analyze this `line 158` and possibly `exit 1`.

Reply:
Agree, the `tee` needs to go here too.

Reply:
Ah, yes that makes sense. Thanks!